flexget/plugins/list/trakt_list.py

```diff
@@ -247,7 +247,8 @@ def invalidate_cache(self):
     def get_list_endpoint(self, remove=False, submit=False):
         # Api restriction, but we could easily extract season and episode info from the 'shows' type
         if not submit and self.config['list'] in ['collection', 'watched'] and self.config['type'] == 'episodes':
-            raise plugin.PluginError('`type` cannot be `%s` for %s list.' % (self.config['type'], self.config['list']))
+            endpoint = ('sync', 'history', 'episodes')
+            return endpoint
```

Comment:
You can just combine this to one line `return ...`. This change is also a bit too broad as Trakt still doesn't have an endpoint for collected episodes.

Reply:
Thanks for checking. I added the additional check and the redundant variable assignment.